common/src/app/common/files/libraries_helpers.cljc

```diff
@@ -68,7 +68,7 @@
                                                        (map :id shapes)
                                                        nil
                                                        root-name
-                                                       true))]
```

Comment:
I'm not sure this may have undesired side effects.

Have you checked all places where `generate-add-component` is called from? And this parameter is ok being changed to false in all cases?